.github/actions/cancel-workflow/action.yml

```diff
@@ -23,5 +23,5 @@ runs:
       shell: pwsh
       run: |
         Write-Output "::error::Canceling workflow due to one of the jobs failing."
-        $url = "https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/cancel"
-        Invoke-WebRequest $url -Headers (Get-GitHubApiAuthorizationHeader) -Method Post
+        # See https://cli.github.com/manual/gh_run_cancel
+        gh run cancel --repo ${{ github.repository }} ${{ github.run_id }}
```

Comment:
Test this with a typo that Spelling notices, for example.

Reply:
Cancellation like this. Looks like another advantage of using `gh` instead of `Invoke-WebRequest`, this only writes "✓ Request to cancel workflow submitted." to the log instead of dumping the whole web request object.

![image](https://user-images.githubusercontent.com/4007293/209891159-84b2d97d-4035-414b-a352-dedeed0ee373.png)